# 单个水果的检测信息
int32 type        # 水果类型 (1-12)
float32 offset_x  # X轴偏移量
float32 offset_y  # Y轴偏移量
bool isgood       # 是否为好水果

